native/src/modules/common/components/ProgressBar.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import * as React from 'react'
+import Svg, { Circle, Image } from 'react-native-svg'
+import { Dimensions } from 'react-native'
+import styled from 'styled-components/native'
+import { type StyledComponent } from 'styled-components'
+import { buildConfigAssets } from '../../app/constants/buildConfig'
+import type { ThemeType } from '../../theme/constants'
+import { type TFunction } from 'react-i18next'
+
+const Container: StyledComponent<{||}, {||}, *> = styled.View`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const Text = styled.Text`
+  padding-top: 25px;
+  font-size: 20px;
+  font-weight: 700;
+`
+
+const { width } = Dimensions.get('window')
+const size = width / 3 // eslint-disable-line no-magic-numbers
+
+const logoSize = size * 0.6 // eslint-disable-line no-magic-numbers
+const logoXY = (size - logoSize) / 2
+const logoToCenter = `translate(${logoXY}, ${logoXY})`
+
+// progress
+const strokeWidth = size * 0.09 // eslint-disable-line no-magic-numbers
+const r = (size - strokeWidth) / 2
+const cx = size / 2
+const cy = size / 2
+const circumference = r * 2 * Math.PI
+const strokeDasharray = circumference
```

Comment:
All of these can be inlined in the component below. That way it is maybe more readable and there are less variables with global scope.

Reply:
done